.circleci/test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script runs all tests in glow, including onnxifi gtests
+
+
+set -euxo pipefail
+
+GLOW_DIR=$PWD
+GLOW_BUILD_DIR=${GLOW_DIR}/build
+cd ${GLOW_BUILD_DIR}
+
+TEST_NAME=$CIRCLE_JOB
+
+# Pass in which tests to run (one of {test, test_unopt}).
+run_unit_tests() {
+    CTEST_PARALLEL_LEVEL=4 ninja ${1} || ( cat Testing/Temporary/LastTest.log && exit 1 )
```

Comment:
noob question: do we have all hosts with 4 cpus?

Reply:
I am not very sure about this... Do we have it in travis?

Reply:
Travis hosts have 2 CPUs, so we used 2 threads.